src/runtime/entitlements-manager-test.js

```diff
@@ -377,7 +438,10 @@ describes.realWin('EntitlementsManager', {}, (env) => {
         );
       expectLog(AnalyticsEvent.ACTION_GET_ENTITLEMENTS, false);
       expectLog(AnalyticsEvent.EVENT_UNLOCKED_BY_SUBSCRIPTION, false);
-
+      /*expectEntitlementPingback(
+        EntitlementSource.GOOGLE_SUBSCRIBER_ENTITLEMENT,
+        EntitlementResult.UNLOCKED_SUBSCRIBER
+      ); TODO: WHY NOT?*/
```

Comment:
Old code or still valid?

Reply:
It was because events are going to the event manager mock and not being sent along from there (which is fine).  We are testing the listening behavior elsewhere.